articles/node-vs-nuxt/index.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/node-vs-nuxt/
+title: Node vs Nuxt - The Key Differences
+description: A comparison betweeen Node.js and Nuxt, two of the most popular web technologies.
+author: saiharsha-balasubramaniam
+date: 2020-09-06T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/node-eslint/hero.jpg
+    alt: linters example image ESLint
+---
+
+A while ago, traditional server side rendering used to be the norm. All of the HTML is dynamically rendered on the server and sent to the client. Back in those days, websites weren't user-friendly and native.
+
+Today's web consists of single-page applications and universal web-apps. Single-page applications are applications that does not reload the entire page every time we click on a link within the browser. It bundles and loads the entire JavaScript onto the client. It then renders parts of the webpage on the client-side, resulting in a fluid user-experience.
+
+Universal applications have parts of the application already rendered by the server. This reduces initial loading times and improves Search Engine Optimization.
+
+Let us look at the differences between Node.js and Nuxt which are two important technologies used for the web.
```

Comment:
Consider revising to two important **technologies that use server-side rendering and client-side rendering.**